### PR TITLE
Subfolders for component styles

### PR DESCRIPTION
**Pull request by hakubo**
## Description

This way our styles folder get at least a bit more manageable

## Reviewers

@Wikia/x-wing @Wikia/social-frontend

### CODE REVIEW

File: front/main/app/styles/component/discussion/index.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+@import 'discussion-collapsable';
+@import 'discussion-sort';
+@import 'discussion-hero-unit';
+@import 'discussion-error';
+@import 'discussion-dialog';
+@import 'discussion-list-modal';
+@import 'discussion-open-graph';
+@import 'discussion-standalone-editor';
+@import 'discussion-editor-overlay-message';
+@import 'user-avatar';
```
Comment (rogatty) [inline front/main/app/styles/component/discussion/index.scss]: :speak_no_evil:

**Comment by tomasznapieralski** [comment]
LGTM

**Comment by kvas-damian** [comment]
LGTM

**Comment by rogatty** [comment]
LGTM

**Comment by bkoval** [comment]
LGTM!

**Pull request merged**